recipes/libiconv/all/conanfile.py

```diff
@@ -24,7 +24,7 @@
 class LibiconvConan(ConanFile):
     name = "libiconv"
     description = "Convert text to and from Unicode"
-    license = "LGPL-2.1"
+    license = "LGPL-2.1-or-later"
```

Comment:
https://git.savannah.gnu.org/gitweb/?p=libiconv.git;a=blob;f=include/iconv.h.in;h=62f7b351a3907ec85df39679b2803460425487b0;hb=HEAD#l7 👍

Reply:
Sure thing!